Purchases/Public/RCPurchases.h

```diff
@@ -375,6 +375,26 @@ NS_SWIFT_NAME(restoreTransactions(_:));
            withDiscount:(SKPaymentDiscount *)discount
         completionBlock:(RCPurchaseCompletedBlock)completion NS_SWIFT_NAME(purchasePackage(_:discount:_:)) API_AVAILABLE(ios(12.2), macosx(10.14.4));
 
+
+/**
+ Invalidates the cache for purchaser information.
+ This is useful for cases where purchaser information might have been updated outside of the app, like if a
+ promotional subscription is granted through the RevenueCat dashboard.
+ */
+- (void)invalidatePurchaserInfoCache;
+
+#pragma mark Subscriber Attributes
+
+- (void)setAttributes:(NSDictionary<NSString *, NSString *> *)attributes;
```

Comment:
NSDictionaries are dumb: 
> Neither a key nor a value can be nil; if you need to represent a null value in a dictionary, you should use NSNull.

https://developer.apple.com/documentation/foundation/nsdictionary?language=objc

You can technically use `NSNull`, but since `NSNull` != `NSString`, it'd kill the advantages (mostly compiler warnings) of specifiying the type for the dict.

Reply:
also, the compiler doesn't let you specify something like
```
NSDictionary<NSString *, NSString * _nullable > *
```